README.md

```diff
@@ -13,6 +35,24 @@ Set `'ENGINE': 'django_tidb'` in your settings.
 - mysqlclient 2.0.3
 - Python 3.6 an newer (tested with Python 3.8)
 
+## Testing
+
+create your virtualenv with:
+
+```
+$ virtualenv venv
+$ source venv/bin/activate
+```
+
+you can use the command ```deactivate``` to exit from the virtual environment.
+
+run all integration tests.
+
+```
+$ python run_testing_worker.py
+```
+
 ## Known issues
```

Comment:
I suggest to add the related issue here:

- TiDB does not support Savepoint([#6840](https://github.com/pingcap/tidb/issues/6840)).